### PR TITLE
Build maker extended

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

**Comment by bgruening** [comment]
It seems that it is touching all files and tries to read the metadata :(

**Comment by mvdbeek** [comment]
Both errors should be fixed by https://github.com/galaxyproject/galaxy/pull/9228

**Comment by mvdbeek** [comment]
Alright, so with https://github.com/galaxyproject/galaxy/pull/9228 it passes

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ before_install:
 
 script:
 - set -e
-- mulled-build-files --check-published --singularity --namespace $MULLED_NAMESPACE build-and-test ./combinations --verbose
+- DEST_BASE_IMAGE="bgruening/busybox-bash:0.1" mulled-build-files --check-published --singularity --namespace $MULLED_NAMESPACE build-and-test ./combinations --verbose
```
Comment (bgruening) [inline .travis.yml]: @mvdbeek why is this needed? Can we make this the default as it was before? Thinking about people that use the script on the command line and our training material.

Comment (mvdbeek) [reply .travis.yml]: Sure, this isn't needed after https://github.com/galaxyproject/galaxy/pull/9228, but mulled-build will figure out the base image automatically if we don't set this ... and we probably don't want this because then we wouldn't necessarily have a match between the tsv files and what we actually build. Let me see if we can default this in the script.

File: .travis.yml [hunk 2]
```diff
@@ -49,7 +49,7 @@ after_success:
     eval $(ssh-agent)
     sshpass -e -P 'Enter passphrase for' ssh-add ~/.ssh/galaxy_depot_travis
 
-     mulled-build-files --check-published --namespace $MULLED_NAMESPACE --oauth-token $QUAY_OAUTH_TOKEN push ./combinations;
+     DEST_BASE_IMAGE="bgruening/busybox-bash:0.1" mulled-build-files --check-published --namespace $MULLED_NAMESPACE --oauth-token $QUAY_OAUTH_TOKEN push ./combinations;
```
Comment (bgruening) [inline .travis.yml]: this one can also go I think ...

**Comment by bgruening** [comment]
Ok! Lets get this in.
